packages/cinterop/src/jvmMain/linux/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.4.1)
```

Comment:
Add a comment here about changes to this also needs to happen in the other files